@xen-orchestra/backups/_VmBackup.js

```diff
@@ -251,6 +255,11 @@ class VmBackup {
       Task.info('Transfer data using NBD')
     }
     const sizeContainers = mapValues(deltaExport.streams, stream => watchStreamSize(stream))
+
+    if (this._settings.validateVhdStreams) {
+      deltaExport.streams = mapValues(deltaExport.streams, stream => pipeline(stream, vhdStreamValidator, noop))
```

Comment:
I think it may remove the `_nbd` property used to show in the UI that nbd is used . Maybe we should put this property in the VM object ?

(previous changes like throttle may already have the same effect)

Reply:
If I understand correctly, the NBD test is before, therefore this change does not break it.